source/elements/oneMKL/source/domains/blas/asum.rst

```diff
@@ -58,15 +49,27 @@ asum
    where ``x`` is a vector with ``n`` elements.
 
 
+asum (BUFFER Version)
+---------------------
+
+.. container::
```

Comment:
The .. container is an artifact of the translation from the original doc. It doesn't do anything. It's ok to commit this now, but we should do a cleanup pass (at the same time we have to unindent the contents). Same for :class: sectiontitle

Reply:
Agree, I will add it to our todo list

Reply:
I think we have slowly come to understand the same thing (that .. container does not do anything)  We plan to clean this up at some point, but as you say, it will require lots of indentation changes...